tests/tests/test_unit.py

```diff
@@ -152,17 +159,104 @@ def test_radd_sync(monkeypatch):
     )
 
     monkeypatch.setattr(
-        RADDAlertsSync, "_get_latest_api_version", lambda x: "v20211018"
+        DeforestationAlertsSync, "get_latest_api_version", lambda x, y: "v2021018"
```

Comment:
I think you lost a character, shouldn't this be
v20211018
instead of 
v2021018

Reply:
oh you're right. I think it doesn't affect much because it's just comparing alphanumerically, but I'll fix it